hyperspy/conftest.py

```diff
@@ -1,12 +1,17 @@
 # Configure mpl and traits to work in a headless system
 from traits.etsconfig.api import ETSConfig
 ETSConfig.toolkit = "null"
-import matplotlib
-matplotlib.use("Agg")
+
+# pytest-mpl 0.7 already import pyplot this line is called, so setting the 
+# matplotlib backend to 'agg' as early as we can will be useless.
+# However, resetting the rcParams to matplotlib default does the job.
+import matplotlib.pyplot as plt
+plt.rcParams.clear()
```

Comment:
I think that locally we should test using the default mpl backend in the system. Otherwise [this test](https://github.com/hyperspy/hyperspy/blob/RELEASE_next_minor/hyperspy/tests/drawing/test_utils.py#L33) will always be skipped.